x/mongo/driver/topology/rtt_monitor.go

```diff
@@ -249,26 +251,57 @@ func percentile(perc float64, samples []time.Duration, minSamples int) time.Dura
 	return time.Duration(p)
 }
 
-// getRTT returns the exponentially weighted moving average observed round-trip time.
-func (r *rttMonitor) getRTT() time.Duration {
+// EWMARTT returns the exponentially weighted moving average observed round-trip time.
+func (r *rttMonitor) EWMARTT() time.Duration {
```

Comment:
Don't love the name `EWMARTT` for this method... Let me know if you have any better ideas.

Reply:
Mathematically, "moving average is a form of a **convolution**" if you think _convolution_ is clear enough.

Reply:
Ah good idea. I think instead I'll just remove `RTT` from the method names: now `RTTMonitor` contains `EWMA()`, `Min()` and `P90()` (thanks for idea offline @matthewdale).